deploy/chart/templates/0000_50_olm_02-services.yaml

```diff
@@ -31,9 +31,48 @@ spec:
   type: ClusterIP
   ports:
   - name: https-metrics
-    port: 8081
+    port: {{ .Values.catalog.service.internalPort }}
     protocol: TCP
-    targetPort: metrics
+    targetPort: {{ .Values.catalog.service.externalPort }}
+  selector:
+    app: catalog-operator
+{{ end }}
+{{ if .Values.pprof.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: olm-operator-pprof
+  namespace: {{ .Values.namespace }}
+  annotations:
+    service.alpha.openshift.io/serving-cert-secret-name: olm-operator-serving-cert
```

Comment:
Just a quick note: I believe this annotation has graduated to beta when I was implementing a downstream solution for the PackageServer CSV to be deployed as a deployment/APIService/etc. set of resources.

Reply:
(Ah, nevermind didn't read the conversation about these manifests being removed -- disregard).